### PR TITLE
remove shared lib

### PR DESCRIPTION
**Pull request by sorenoid**
Related to issue: https://devtopia.esri.com/runtime/kotlin/issues/4897

### Description:

remove shared lib usage from popup and UN. Since the shared lib won't be published, it cannot be specified as a gradle dependency.

### Summary of changes:

- move ExpandableCard code into UN, Popup. Move remaining AOSP code into UN. Fix up gradle.

### Pre-merge Checklist

- a [vTest](https://runtime-kotlin.esri.com/view/all/job/vtest/job/toolkit/) Job for this PR has been run
  - [x] link: https://runtime-kotlin.esri.com/view/vTest/job/vtest/job/toolkit/195/
- Unit and/or integration tests have been added to exercise this PR's logic, and the tests are passing:
  - [ ] Yes
  - [ ] No

### CODE REVIEW

File: toolkit/utilitynetworks/src/main/java/com/arcgismaps/toolkit/utilitynetworks/ui/expandablecard/ExpandableCard.kt
Comment (eri9000) [inline toolkit/utilitynetworks/src/main/java/com/arcgismaps/toolkit/utilitynetworks/ui/expandablecard/ExpandableCard.kt:1]: To follow Popup, should this go inside a folder called `internal`?

**Review by eri9000 — APPROVED** [review]
One suggestion, otherwise LGTM 👍🏼

File: toolkit/utilitynetworks/src/main/java/com/arcgismaps/toolkit/utilitynetworks/ui/expandablecard/ExpandableCard.kt
Comment (sorenoid) [reply toolkit/utilitynetworks/src/main/java/com/arcgismaps/toolkit/utilitynetworks/ui/expandablecard/ExpandableCard.kt:1]: I don't see a reason to change the folder structure of the project.

**Review by shubham7109 — APPROVED** [review]
Looks good, tested by generating the dokka and these changes are not visible in the package declarations.

**Pull request merged**